modules/server/src/network/gql.ts

```diff
@@ -0,0 +1,18 @@
+import { print } from 'graphql';
+
+export const createGqlQuery = ({ query: queryAST, variables }) => {
+	// queries need to be strings when using regular http
+	const query = print(queryAST);
+	return JSON.stringify({ query, variables });
```

Comment:
no validation needed? or is that ToDo?

Reply:
the actual queries will be written when we write the resolvers
this function is just to go from gqlObject to string request for gql
made more sense to keep it in for future usage

Reply:
originally I was creating handmade queries to get the schemas, but I found an inbuilt gql method to get an introspection query